src/Components/Web/src/Forms/InputSelect.cs

```diff
@@ -24,16 +27,51 @@ public class InputSelect<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTy
         /// </summary>
         [DisallowNull] public ElementReference? Element { get; protected set; }
 
+        /// <summary>
+        /// Gets or sets the current value of the input as a string array.
+        /// </summary>
+        protected string?[]? CurrentValueAsStringArray
```

Comment:
Can we make this `private`?

Reply:
I suppose it depends on whether we imagine someone is going to subclass `InputSelect` and provide their own formatting/parsing logic. That would not be an outrageous thing to do, so maybe we should leave this as `protected`.

Reply:
Thinking about it more, we don't really support providing completely custom formatting/parsing logic for arrays anyway, since there's hardcoded logic in other places that requires the value to be formatted as a JSON array.

Unless you can think of definite scenarios where someone would be trying to customize this, could we err on the side of making this `private` for now? We can always make it into public API in the future if we want.

Reply:
Sure, I'll change it to `private`. My concern was indeed subclassing `InputSelect` as I recall others were trying to do that at some point. But I agree, seems better to make it `private` until we see a need to change it.